client/src/app/modules/module-instance/components/fields/chips/chips.component.ts

```diff
@@ -27,8 +28,12 @@ export class ChipsComponent extends FieldComponent<ChipsData>
     const input = event.input;
 
     if ((value || '').trim()) {
-      this.data.push(value.trim());
-      this.cData.control.setValue(this.data);
+      if (this.data.includes(value.trim()) && this.cData.unique === true) {
```

Comment:
Also, this condition should be flipped around so that we don't have an additional unnecessary block. `this.cData.unique === true` - you don't need to compare with `true`